Core/GDCore/IDE/Project/ObjectsUsingResourceCollector.cpp

```diff
@@ -0,0 +1,19 @@
+#include "ObjectsUsingResourceCollector.h"
+
+#include "GDCore/Project/Object.h"
+#include "GDCore/Project/Project.h"
+
+namespace gd {
+
+void ObjectsUsingResourceCollector::DoVisitObject(gd::Object& object) {
+  gd::ResourceNameMatcher resourceNameMatcher = gd::ResourceNameMatcher(resourceName);
```

Comment:
Ubernit (or use `auto` to save typing the class name twice):
```suggestion
  gd::ResourceNameMatcher resourceNameMatcher(resourceName);
```